setup.py

```diff
@@ -12,7 +12,10 @@
     author="Algorand",
     author_email="pypiservice@algorand.com",
     python_requires=">=3.9",
-    install_requires=["py-algorand-sdk>=2.0.0", "tabulate==0.9.0"],
+    install_requires=[
+        "py-algorand-sdk@git+https://github.com/algorand/py-algorand-sdk@better-exports",
+        "tabulate==0.9.0",
+    ],
```

Comment:
```suggestion
    install_requires=["py-algorand-sdk>=2.0.1", "tabulate==0.9.0"],
```
^^^ TODO: should be something like this in anticipation of the PR merge and version update ^^^